### PR TITLE
Hide MMR on profile if less than 10 matches

### PR DESCRIPTION
**Pull request by bjarkehs**
image

### CODE REVIEW

**Comment by lajtmaN** [comment]
Remove it from the "quick-view" modal of a player as well

**Comment by lajtmaN** [comment]
Or maybe we should show the 🐥 instead of hiding MMR completely

File: backend/controllers/stats_controller.go [hunk 1]
```diff
@@ -74,9 +75,22 @@ func (sc StatsController) GetPlayerHistory(c *gin.Context) {
 		return
 	}
 
+	userCount := make(map[uint]int)
+	for _, entry := range entries {
+		userCount[entry.UserID]++
+	}
```
Comment (lajtmaN) [inline backend/controllers/stats_controller.go]: This confuses me a bit - isn't it `matchCount`?

File: frontend/src/routes/(authed)/player/[id]/+page.server.ts [hunk 2]
```diff
@@ -42,7 +42,8 @@ export const load: PageServerLoad = async ({
   const lost = totalMatches - wins;
   const winrate = totalMatches > 0 ? wins / totalMatches : 0;
 
-  const mmr = mmrHistory[mmrHistory.length - 1]?.mmr ?? null;
+  const mmr =
+    totalMatches > 10 ? mmrHistory[mmrHistory.length - 1]?.mmr ?? null : null;
```
Comment (lajtmaN) [inline frontend/src/routes/(authed)/player/[id]/+page.server.ts]: Do you need this after adding the filter in backend?

Comment (bjarkehs) [reply frontend/src/routes/(authed)/player/[id]/+page.server.ts]: No, I'll refactor this.

Comment (bjarkehs) [reply frontend/src/routes/(authed)/player/[id]/+page.server.ts]: Fixed

**Review by lajtmaN — APPROVED** [review]

**Pull request merged**